server/prometheus.js

```diff
@@ -20,6 +20,31 @@ const monitorCertIsValid = new PrometheusClient.Gauge({
     help: "Is the certificate still valid? (1 = Yes, 0= No)",
     labelNames: commonLabels
 });
+
+const monitorUptime1y = new PrometheusClient.Gauge({
```

Comment:
please go through https://prometheus.io/docs/practices/naming/ and make sure that the new metrics are better "alligned" to what prometheus wants metrics to be like ^^